core/worker.go

```diff
@@ -614,6 +653,106 @@ func (w *worker) GeneratePendingHeader(block *types.WorkObject, fill bool) (*typ
 	return newWo, nil
 }
 
+func (w *worker) OrderTransactionSet(txs []*types.Transaction, gasUsedAfterTransaction []uint64, block *types.WorkObject) {
```

Comment:
Please add a comment for this function, I'm guessing that this function is the ideal revenue/gas price pick? So initially we pick the non-ideal gas price and here we pick the ideal?

Reply:
Yeah, added detailed comments about the function